src/dtos/unit.dto.ts

```diff
@@ -93,19 +100,110 @@ export class UnitDTO {
   endDate: Date;
 
   @ApiProperty({
-    description:
-      propertyMetadata.monitorLocationDTONonLoadBasedIndicator.description,
-    example: propertyMetadata.monitorLocationDTONonLoadBasedIndicator.example,
-    name:
-      propertyMetadata.monitorLocationDTONonLoadBasedIndicator.fieldLabels
-        .value,
+    description: propertyMetadata.unitDTOSourceCategoryCd.description,
+    example: propertyMetadata.unitDTOSourceCategoryCd.example,
+    name: propertyMetadata.unitDTOSourceCategoryCd.fieldLabels.value,
   })
   @IsOptional()
-  @IsInt()
-  @IsInRange(0, 1, {
-    message: (args: ValidationArguments) => {
-      return `The value of [${args.value}] for [${args.property}] must be an integer of 0 and 1 for [${KEY}].`;
-    },
+  @IsString()
+  sourceCategoryCd?: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOCommOpDate.description,
+    example: propertyMetadata.unitDTOCommOpDate.example,
+    name: propertyMetadata.unitDTOCommOpDate.fieldLabels.value,
   })
-  nonLoadBasedIndicator: number;
+  @IsOptional()
+  @IsDateString()
+  commOpDate?: string;
```

Comment:
Why is this date a string? Was COMM_OP_DATE confused with COMM_OP_DATA_CD?

Reply:
This is explicitly cast to a string in the format `yyyy-mm-dd` in the `getUnitDetails` function of `UnitService` and `UnitWorkspaceService`.